homework-submission/react-hyf/react-hyf-client/src/App.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import 'bootstrap/dist/css/bootstrap.css';
+
+import HomeworkWeek1 from './components/homeworkweek1/homeworkweek1';
+import HomeworkWeek2 from './components/homeworkweek2/homeworkweek2';
+import HomeworkWeek3 from './components/homeworkweek3/homeworkweek3';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="App-header">
+        <h2>HomeworkWeek1</h2>
+        <HomeworkWeek1 />
+        <h2>HomeworkWeek2</h2>
+        <HomeworkWeek2 />
+        <h2>HomeworkWeek3</h2>
+        <div className="container week3">
+          <table className="table table-dark">
```

Comment:
perfect tabling, making it better by customizing it via assigning it into a component, and using the children `prop`:
```javascript
<table className="table table-dark">
  <thead>
    <tr>
      <th scope="col">#</th>
      <th scope="col">Name</th>
      <th scope="col">Surname</th>
      <th scope="col">Gender</th>
      <th scope="col">Region</th>
    </tr>
  </thead>
  <tbody>
    {this.props.children}
    {/* this will be the inserted value of <Comp>child</Comp> and it will have the exact same css as you have in the parent */}
  </tbody>
</table>
```